### PR TITLE
Fix misleading VOD message

### LINKED ISSUE #4987: Downloads always start in VOD mode
Every download seems to start in VOD mode. This is reflected in the log, and can be seen as well in the download pieces map prioritizing the beginning of the torrent. This reduces download performance.

Instead, VOD should be used **only** if the user clicked the "Play" button.

```python
INFO    1575206079.86             LibtorrentMgr:671  (LibtorrentMgr)  start_download: Starting in VOD mode
```

### PR DESCRIPTION
**Pull request by ichorid**
Fixes #4987

### CODE REVIEW

File: Tribler/Core/Libtorrent/LibtorrentMgr.py [hunk 1]
```diff
@@ -666,9 +666,7 @@ def start_download(self, torrentfilename=None, infohash=None, tdef=None, dconfig
                 d.get_def().get_trackers_as_single_tuple()))
             if new_trackers:
                 self.tribler_session.update_trackers(tdef.get_infohash(), new_trackers)
-            return d
```
Comment (qstokkink) [inline Tribler/Core/Libtorrent/LibtorrentMgr.py]: Was this removal intentional?

Comment (ichorid) [reply Tribler/Core/Libtorrent/LibtorrentMgr.py]: oops. It slipped from another PR. That's why we do reviews...
